lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -46,6 +47,8 @@ class BraintreeBlueGateway < Gateway
         cannot_refund_if_unsettled: 91506
       }
 
+      DIRECT_BANK_ERROR = 'Direct bank account transactions are not supported. Bank accounts must be stored and verified before use.'.freeze
```

Comment:
```suggestion
      DIRECT_BANK_ERROR = 'Direct bank account transactions are not supported. Bank accounts must be successfully stored before use.'.freeze
```
Slightly more useful message, as verificatino could be taken to mean running a verify action, when the verification is built into the store action.